src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md

```diff
@@ -0,0 +1,148 @@
+metrics: [LCP](https://web.dev/lcp/#what-is-lcp), [FID](https://web.dev/fid/#what-is-fid), and
+[CLS](https://web.dev/cls/#what-is-a-good-cls-score). So for example, the CrUX API can tell us that
+16% of real-user experiences on a given page are evaluated as having good
+<span style="white-space: nowrap;">LCP ≤ 2.5</span> seconds.
+
+<figure class="w-figure">
+{% Img src="image/STd8eW8CSiNp5B1bX0R6Dww2eH32/EaVNEuQ6gxVLtHYNkSZj.png", alt="Screenshot of the
+Web Vitals extension showing an explanation of how the local LCP experience relates to real-user
+desktop data from the field.", width="800", height="549" %}
+  <figcaption class="w-figcaption">
+    Screenshot of the Web Vitals extension showing an explanation of how the local LCP experience
+    relates to real-user desktop data from the field.
+  </figcaption>
+</figure>
+
+The way this is integrated with the Web Vitals extension is by layering your local experience on
+top of the broader distribution of real-user experiences. It's like when you're in an expansive
+place like a shopping mall and there's a big map with all of the stores and a pin labeled
+"YOU ARE HERE" to help you orient yourself and find where you're going. The distributions of
+real-user data from CrUX are laid out for each metric and your local experiences are horizontally
+positioned in such a way that indicates where your experiences fall in relation to the
+distributions. Something that should be immediately apparent from this UI is when your local
+experience is very different from other users. For example, if you have a slow LCP experience and
+only 1% of users have similar experiences, something unusual must have happened.
+
+<figure class="w-figure">
+{% Img src="image/STd8eW8CSiNp5B1bX0R6Dww2eH32/t5SmXUJqJadNsyOi26bb.png", alt="Screenshot of the
+Web Vitals extension showing origin-level desktop field data with \"Waiting for input…\" for the
+FID results.", width="800", height="552" %}
+  <figcaption class="w-figcaption">
+    Screenshot of the Web Vitals extension showing origin-level desktop field data with
+    "Waiting for input…" for the FID results.
+  </figcaption>
+</figure>
+
+It's common to visit a page that may have no URL-level data in the CrUX dataset. This can happen
+because the page is so new that it hasn't been picked up yet or because the page has insufficient
+traffic to be included in the dataset. In cases like these, we attempt to fall back to more
+granular origin-level data whenever possible. This data represents the aggregate user experiences
+on all pages of the website, so while it may not be immediately relevant to your particular page
+experience, it should still offer some insight into how users experience the site as a whole.
+Another common case is to see a "Waiting for FID" message. This is because FID is the only
+Core Web Vital that requires user interaction to be measured, so we won't have any local FID data
+to show until you interact with the page.
```

Comment:
```suggestion
Some pages don't have URL-level data in the CrUX dataset. This can happen
because the page is so new that it hasn't been picked up yet or because the page has insufficient
traffic to be included in the dataset. In cases like these, Web Vitals extension falls back to more
granular origin-level data whenever possible. This data represents the aggregate user experiences
on all pages of the website, so while it may not be immediately relevant to your particular page
experience, it should still offer some insight into how users experience the site as a whole.

Another common case is to see a "Waiting for input.." message. This is because FID is the only
Core Web Vital that requires user interaction to be measured, so the extension won't have any local FID data
to show until you interact with the page.
```

Reply:
The UI shows "Waiting for input..." message in my version so I've edited that.